File: NERP.NetworkEntities/Models/Splitter.cs
using NERP.Common.Models;

namespace NERP.NetworkEntities.Models
{
    public class Splitter : BaseNetworkEntity
    {
        public Point Geometry { get; set; }
        public string SplitterRatio { get; set; }
        public double? LossValue { get; set; }
        public Optics Optics { get; set; }
        public Installation Installation { get; set; }
        public Owner Owner { get; set; }
    }
}

File: NERP.Admin.Web/ClientApp/src/app/entities/models/splitter.ts
import { Point } from '../../common/geo/geometry';
import { BaseNetworkEntity } from './baseNetworkEntity';
import { Installation } from './installation';
import { Optics } from './optics';
import { Owner } from './owner';

export class Splitter extends BaseNetworkEntity {
    splitterRatio?: string;
    lossValue?: number;
    optics?: Optics = new Optics();

    geometry: Point;
    installation?: Installation = new Installation();
    owner?: Owner = new Owner();
}
